CONTRIBUTING.md

```diff
@@ -32,33 +20,70 @@ The following is a set of guidelines for contributing to Redash. These are guide
 - [Release Method](#release-method)
 - [Code of Conduct](#code-of-conduct)
 
+## Quick Links:
+
+- [Feature Requests](https://discuss.redash.io/c/feature-requests)
+- [Documentation](https://redash.io/help/)
+- [Blog](https://blog.redash.io/)
+- [Twitter](https://twitter.com/getredash)
```

Comment:
I would replace these two with the forum.

Reply:
Fixed [3edc967](https://github.com/getredash/redash/pull/5714/commits/3edc967ba74b3b826975e61433c2cd5311de90a9)